service/pom.xml

```diff
@@ -221,6 +221,10 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
+      <groupId>org.springframework</groupId>
+      <artifactId>spring-core</artifactId>
```

Comment:
why is this change?

Reply:
Spring core is required for exception handling. Spring converts SqlException to DataAccessException (which is runtime not checked). During the conversion, based on the SQL codes for some exceptions (but not for all) spring also localizes the exception message. This is the point where spring-core is required. It is not used directly at all, no spring DI introduced in this PR

Reply:
spring-core is already added to metastore-server, why it's needed here?